src/class-supported-post-types.php

```diff
@@ -63,9 +68,13 @@ public function get_current_post_type(): string|false {
 				$post_id   = absint( $_GET['post'] );
 				$post_type = get_post_type( $post_id );
 			// phpcs:ignore WordPress.VIP.SuperGlobalInputUsage.AccessDetected, WordPress.Security.NonceVerification.NoNonceVerification, WordPress.Security.NonceVerification.Recommended
-			} elseif ( 'post-new.php' === $pagenow && ! empty( $_GET['post_type'] ) ) {
-				// phpcs:ignore WordPress.VIP.SuperGlobalInputUsage.AccessDetected, WordPress.Security.NonceVerification.Recommended
-				$post_type = sanitize_text_field( $_GET['post_type'] );
+			} elseif ( 'post-new.php' === $pagenow ) {
+				if ( ! empty( $_GET['post_type'] ) ) { // phpcs:ignore WordPress.Security.NonceVerification.Recommended
+					$post_type = sanitize_text_field( $_GET['post_type'] ); // phpcs:ignore WordPress.Security.NonceVerification.Recommended
+				} else {
```

Comment:
Curious why we are ignoring nonces and SuperGlobalInputUsage here... would it make sense/be possible to use a nonce?